apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/template-selector-item.js

```diff
@@ -31,6 +30,13 @@ const TemplateSelectorItem = ( props ) => {
 		return null;
 	}
 
+	const staticPreviewImg =
+		'blank' === value
+			? null
+			: 'https://s0.wordpress.com/mshots/v1/' +
+			  encodeURI( 'https://dotcompatterns.wordpress.com/' + value ) +
```

Comment:
Instead of using the source site directly for the preview, we can use the `template/demo` endpoint like in: https://github.com/automattic/wp-calypso/blob/trunk/client/landing/gutenboarding/available-designs.ts#L44

In this endpoint, if we pass in `post_id`, it'll hide the header and footer of the site and just render the content of the pattern. E.g. https://public-api.wordpress.com/rest/v1/template/demo/hever/dotcompatterns.wordpress.com/?post_id=1340&language=en

It also already uses the Patterns Toolkit to handle translations, so as another example here's a pattern in Spanish (note that it looks like the main heading doesn't have a translation for that string yet, but everything else in this pattern appears to be translated): https://public-api.wordpress.com/rest/v1/template/demo/hever/dotcompatterns.wordpress.com/?post_id=1378&language=es

Reply:
Thanks! Now updated to use the template/demo endpoint and use the locale.